app/helpers/ticketing.py

```diff
@@ -291,26 +291,6 @@ def initiate_order_payment(form, paid_via=None):
         order = TicketingManager.get_and_set_expiry(identifier)
 
         if order:
-
-            holders_firstnames = form.getlist('holders[firstname]')
-            holders_lastnames = form.getlist('holders[lastname]')
-            holders_ticket_ids = form.getlist('holders[ticket_id]')
-            holders_emails = form.getlist('holders[email]')
-
-            for i, firstname in enumerate(holders_firstnames):
-                data = {
-                    'firstname': firstname,
-                    'lastname': holders_lastnames[i]
-                }
-                holder_user = TicketingManager.get_or_create_user_by_email(holders_emails[i], data)
-                ticket_holder = TicketHolder(firstname=data['firstname'],
-                                             lastname=data['lastname'],
-                                             ticket_id=int(holders_ticket_ids[i]),
-                                             email=holder_user.email,
-                                             order_id=order.id)
-                DataManager.add_attendee_role_to_event(holder_user, order.event_id)
-                db.session.add(ticket_holder)
-
             user = TicketingManager.get_or_create_user_by_email(email, form)
```

Comment:
@shubham-padia, this was moved up to ensure that even if payment is not completed but the Pay Now button is clicked, then attendees are added to the DB. So the attendees are added to the DB with a initiated status even when payment isn't completed. Can you please ensure that this feature still works? Once you ensure this, the remaining seems good to me.